blockly-rtc/test/test.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Unit tests for WorkspaceClient.
+ * @author navil@google.com (Navil Perez)
+ */
+
+const assert = require('assert');
+const sinon = require('sinon');
```

Comment:
Did you add sinon to ``package.json``?

Reply:
Yes I have. I have now uploaded package.json to this change.